app/uk/gov/hmrc/cipphonenumberverification/connectors/ValidateConnector.scala

```diff
@@ -31,10 +30,10 @@ class ValidateConnector @Inject()(httpClientV2: HttpClientV2, config: AppConfig)
 
   val validateUrl = s"${config.validateUrlProtocol}://${config.validateUrlHost}:${config.validateUrlPort}/customer-insight-platform/phone-number/validate"
 
-  def callService(phoneNumber: PhoneNumber)(implicit hc: HeaderCarrier): Future[HttpResponse] = {
+  def callService(phoneNumber: String)(implicit hc: HeaderCarrier): Future[HttpResponse] = {
```

Comment:
why did you change this from PhoneNumber to a string

Reply:
Because we are now using the same function from verify/otp. verify/otp has a PhoneNumberAndOtp object so I changed the signature to take a string so that it will work for both verify and verify/otp